### PR TITLE
Add automation to Wand of Widening

### PR DESCRIPTION
**Pull request by ammalagonc**

### CODE REVIEW

File: packs/feats/widen-spell.json [hunk 1]
```diff
@@ -104,6 +104,79 @@
...
+                        ]
+                    }
+                ],
+                "property": "description",
+                "value": [
+                    {
+                        "text": "PF2E.SpecificRule.Spellshape.WidenSpell.Burst"
+                    }
+                ]
+            },
```
Comment (pedrogrullada) [inline packs/feats/widen-spell.json:128]: Hm I personally don't think we need an addendum if the thing Just Works™, like Widen Spell does.

Comment (ammalagonc) [reply packs/feats/widen-spell.json:128]: So drop the description for everything?

Comment (pedrogrullada) [reply packs/feats/widen-spell.json:128]: Yeah, I think the text being there could make it seem like you need to adjust the size yourself, when in reality it just works

Comment (ammalagonc) [reply packs/feats/widen-spell.json:128]: I like that the pop-up description reminds me that the ability/item has been used on top of the spell.

Would love to have item alterations on action cost to account for this.

File: packs/equipment/wand-of-widening-3rd-rank-spell.json [hunk 2]
```diff
@@ -48,4 +224,4 @@
         }
     },
     "type": "equipment"
-}
+}
```
Comment (pedrogrullada) [inline packs/equipment/wand-of-widening-3rd-rank-spell.json:227]:
```suggestion
}

```

File: packs/equipment/wand-of-widening-4th-rank-spell.json [hunk 3]
```diff
@@ -34,7 +34,183 @@
...
         "quantity": 1,
-        "rules": [],
+        "rules": [
+            {
+                "key": "RollOption",
+                "option": "wand-of-widening",
+                "placement": "spellcasting",
+                "toggleable": true,
+                "label": "{item|name}",
+                "value": true
```
Comment (pedrogrullada) [inline packs/equipment/wand-of-widening-4th-rank-spell.json:44]:
```suggestion
                "toggleable": true
```

File: packs/equipment/wand-of-widening-4th-rank-spell.json [hunk 4]
```diff
@@ -48,4 +224,4 @@
         }
     },
     "type": "equipment"
-}
+}
```
Comment (pedrogrullada) [inline packs/equipment/wand-of-widening-4th-rank-spell.json:227]:
```suggestion
}

```

File: packs/equipment/wand-of-widening-5th-rank-spell.json [hunk 5]
```diff
@@ -34,7 +34,183 @@
...
         "quantity": 1,
-        "rules": [],
+        "rules": [
+            {
+                "key": "RollOption",
+                "option": "wand-of-widening",
+                "placement": "spellcasting",
+                "toggleable": true,
+                "label": "{item|name}",
+                "value": true
```
Comment (pedrogrullada) [inline packs/equipment/wand-of-widening-5th-rank-spell.json:44]:
```suggestion
                "toggleable": true
```

File: packs/equipment/wand-of-widening-5th-rank-spell.json [hunk 6]
```diff
@@ -48,4 +224,4 @@
         }
     },
     "type": "equipment"
-}
+}
```
Comment (pedrogrullada) [inline packs/equipment/wand-of-widening-5th-rank-spell.json:227]:
```suggestion
}

```

File: packs/equipment/wand-of-widening-6th-rank-spell.json [hunk 7]
```diff
@@ -34,7 +34,183 @@
...
         "quantity": 1,
-        "rules": [],
+        "rules": [
+            {
+                "key": "RollOption",
+                "option": "wand-of-widening",
+                "placement": "spellcasting",
+                "toggleable": true,
+                "label": "{item|name}",
+                "value": true
```
Comment (pedrogrullada) [inline packs/equipment/wand-of-widening-6th-rank-spell.json:44]:
```suggestion
                "toggleable": true
```

File: packs/equipment/wand-of-widening-6th-rank-spell.json [hunk 8]
```diff
@@ -48,4 +224,4 @@
         }
     },
     "type": "equipment"
-}
+}
```
Comment (pedrogrullada) [inline packs/equipment/wand-of-widening-6th-rank-spell.json:227]:
```suggestion
}

```

File: packs/equipment/wand-of-widening-7th-rank-spell.json [hunk 9]
```diff
@@ -34,7 +34,183 @@
...
         "quantity": 1,
-        "rules": [],
+        "rules": [
+            {
+                "key": "RollOption",
+                "option": "wand-of-widening",
+                "placement": "spellcasting",
+                "toggleable": true,
+                "label": "{item|name}",
+                "value": true
```
Comment (pedrogrullada) [inline packs/equipment/wand-of-widening-7th-rank-spell.json:44]:
```suggestion
                "toggleable": true
```

File: packs/equipment/wand-of-widening-7th-rank-spell.json [hunk 10]
```diff
@@ -48,4 +224,4 @@
         }
     },
     "type": "equipment"
-}
+}
```
Comment (pedrogrullada) [inline packs/equipment/wand-of-widening-7th-rank-spell.json:227]:
```suggestion
}

```

File: packs/equipment/wand-of-widening-8th-rank-spell.json [hunk 11]
```diff
@@ -34,7 +34,183 @@
...
         "quantity": 1,
-        "rules": [],
+        "rules": [
+            {
+                "key": "RollOption",
+                "option": "wand-of-widening",
+                "placement": "spellcasting",
+                "toggleable": true,
+                "label": "{item|name}",
+                "value": true
```
Comment (pedrogrullada) [inline packs/equipment/wand-of-widening-8th-rank-spell.json:44]:
```suggestion
                "toggleable": true
```

File: packs/equipment/wand-of-widening-8th-rank-spell.json [hunk 12]
```diff
@@ -48,4 +224,4 @@
         }
     },
     "type": "equipment"
-}
+}
```
Comment (pedrogrullada) [inline packs/equipment/wand-of-widening-8th-rank-spell.json:227]:
```suggestion
}

```

File: packs/equipment/wand-of-widening-9th-rank-spell.json [hunk 13]
```diff
@@ -34,7 +34,183 @@
...
         "quantity": 1,
-        "rules": [],
+        "rules": [
+            {
+                "key": "RollOption",
+                "option": "wand-of-widening",
+                "placement": "spellcasting",
+                "toggleable": true,
+                "label": "{item|name}",
+                "value": true
```
Comment (pedrogrullada) [inline packs/equipment/wand-of-widening-9th-rank-spell.json:44]:
```suggestion
                "toggleable": true
```

File: packs/equipment/wand-of-widening-9th-rank-spell.json [hunk 14]
```diff
@@ -48,4 +224,4 @@
         }
     },
     "type": "equipment"
-}
+}
```
Comment (pedrogrullada) [inline packs/equipment/wand-of-widening-9th-rank-spell.json:227]:
```suggestion
}

```

File: packs/feats/widen-spell.json (hunk 1 above)
Comment (pedrogrullada) [reply packs/feats/widen-spell.json:128]: Once that happens, we'd remove those description alterations and have vestigial stuff in the localization file that can't just be deleted without migrating. I don't think they bring enough value to deal with that, specially when the automation works completely.

**Review by pedrogrullada — APPROVED** [review]

**Pull request merged**